pkg/gui/gui.go

```diff
@@ -643,22 +660,9 @@ func (gui *Gui) Run() error {
 		gui.waitForIntro.Add(1)
 	}
 
-	go func() {
-		gui.waitForIntro.Wait()
-		isNew := gui.Config.GetIsNewRepo()
-		if !isNew {
-			time.After(60 * time.Second)
-		}
-		_, err := gui.fetch(g, g.CurrentView(), false)
-		if err != nil && strings.Contains(err.Error(), "exit status 128") && isNew {
-			_ = gui.createConfirmationPanel(g, g.CurrentView(), gui.Tr.SLocalize("NoAutomaticGitFetchTitle"), gui.Tr.SLocalize("NoAutomaticGitFetchBody"), nil, nil)
-		} else {
-			gui.goEvery(time.Second*60, func() error {
-				_, err := gui.fetch(gui.g, gui.g.CurrentView(), false)
-				return err
-			})
-		}
-	}()
+	if gui.Config.GetUserConfig().GetBool("gui.git.autoFetch") {
```

Comment:
we already have a top-level `git` key for the config options so I think this should be "git.autoFetch" and so we'd need to update app_config.go accordingly

Reply:
per `GetDefaultConfig`, `git` is nested under `gui`: https://github.com/jesseduffield/lazygit/blob/master/pkg/config/app_config.go#L248

is that a bug? should the `git` key be a level up?

Reply:
indeed it should. That seems to be an error with just that function, I believe the key is accessed via the top level 'git'